rust/codegen/src/main.rs

```diff
@@ -16,7 +16,6 @@ fn main() {
     prost_build::Config::new()
         .out_dir(&out_dir)
         .format(true)
-        .type_attribute(".", "#[derive(Eq)]")
```

Comment:
What's the reason for removing this?

Reply:
Let's revert this then if @avahowell doesn't object?

Reply:
I had to remove these due to them causing conflicting implementations and breaking the generated code on my machine (test with `cargo test` after running codegen):

```
error[E0119]: conflicting implementations of trait `std::cmp::Eq` for type `ics23::LengthOp`
```